app/controllers/tanks_controller.rb

```diff
@@ -1,6 +1,10 @@
 class TanksController < ApplicationController
   before_action :set_tank, only: [:show]
 
+  def index
+    @tanks = Tank.all
```

Comment:
Similarly with #347, we will want to circle back to scoping the queries in this controller once #336 is completed, but that doesnt need to be a blocker to getting this work merged. we can make a follow up issue.